benchmarks/src/main/scala/zio/redis/benchmarks/lists/BlMoveBenchmarks.scala

```diff
@@ -17,14 +17,13 @@
 package zio.redis.benchmarks.lists
 
 import org.openjdk.jmh.annotations._
-import zio.ZIO
-import zio.duration._
 import zio.redis._
 import zio.redis.benchmarks._
+import zio.{ZIO, _}
```

Comment:
```suggestion
import zio._
```

P.S. please adjust all occurrences.